core/execution/spot/market_snapshot.go

```diff
@@ -199,6 +208,7 @@ func (m *Market) GetState() *types.ExecSpotMarket {
 		Parties:                    parties,
 		Closed:                     m.closed,
 		HasTraded:                  m.hasTraded,
+		FeeStats:                   m.fee.GetState(),
 	}
 
 	println(em.IntoProto().String())
```

Comment:
debug statement? `println` should not be used

Reply:
Thanks, removed now.